src/microbe_stage/editor/OrganellePopupMenu.cs

```diff
@@ -114,10 +123,11 @@ public override void _Ready()
         selectedOrganelleNameLabel = GetNode<Label>(SelectedOrganelleNameLabelPath);
         deleteButton = GetNode<Button>(DeleteButtonPath);
         moveButton = GetNode<Button>(MoveButtonPath);
+        microbeEditor = GetNode<MicrobeEditor>(MicrobeEditorPath);
```

Comment:
Rather than doing this I would 100% more want to see it be so that when this menu is opened the microbe editor needs to be given as a parameter there. This kind of referencing a parent node is not a good design (child object tightly coupled with parent, instead of the parent object injecting the dependency when needed).